src/arviz_stats/loo/compare.py

```diff
@@ -85,6 +90,26 @@ def compare(
            ...: compare_dict = {"non centered": data1, "centered": data2}
            ...: compare(compare_dict)
 
+    Compare models using subsampled LOO:
+
+    .. ipython::  python
+        :okwarning:
+
+        In [1]: from arviz_stats import loo_subsample
+           ...: from arviz_base import load_arviz_data
+           ...: data1 = load_arviz_data("non_centered_eight")
+           ...: data2 = load_arviz_data("centered_eight")
+           ...: loo_sub1 = loo_subsample(data1, observations=6, pointwise=True, seed=42)
+           ...: loo_sub2 = loo_subsample(data2, observations=6, pointwise=True, seed=42)
+           ...: compare({"non_centered": loo_sub1, "centered": loo_sub2})
```

Comment:
The dataframe does not render fully in the example; maybe we could print fewer decimals; the same applies to the previous example.

```suggestion
           ...: compare({"non_centered": loo_sub1, "centered": loo_sub2}).round(2)
```